aviary/validation_cases/benchmark_tests/test_bench_FwGm.py

```diff
@@ -13,11 +12,11 @@
 @use_tempdirs
 class ProblemPhaseTestCase(unittest.TestCase):
 
-    @require_pyoptsparse(optimizer="SNOPT")
```

Comment:
I think we still want to be able to run the SNOPT benches, because we are used to working with and understanding those. I would recommend adding a test that is for IPOPT, but keeping SNOPT as a bench.

Reply:
This occurs in a few places; do you want one SNOPT bench or all of them? The point of keeping them as SNOPT is to better debug them? If we're debugging we could simply set them to IPOPT, right?

Reply:
Ken and I voice-chatted, we landed on:
- keep IPOPT for all benches and tests
- also add SNOPT benches for those with local copies of SNOPT
- don't have SNOPT be a CI dependency

Reply:
If we make optimizer an optional argument to the test that defaults to IPOPT I think we get everything we want. This lets the test suite be runnable by anyone, and for debugging you can call the test from main and select SNOPT (if desired).

Reply:
Thanks Jason, good idea! I saw this after I added IPOPT/SNOPT benches for the swaps. I vote we implement what you suggest later on as needed.